db/models.py

```diff
@@ -1 +1,10 @@
 from django.db import models
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255)
```

Comment:
Char field name, name of the genre with the maximum length of 255 characters, and name must be unique